docs/modules/tutorials/pages/java-client-getting-started.adoc

```diff
@@ -427,7 +427,7 @@ In this tutorial, you learned how to get started with the Hazelcast Java Client,
 == See Also
 
 There are a lot of things that you can do with the Java Client. For more, such as how you can query a map with predicates and SQL,
-check out our https://github.com/hazelcast/hazelcast[Hazelcast repository] and our https://docs.hazelcast.com/hazelcast/5.3/clients/java#hide-nav[Java client documentation] to better understand what is possible.
+check out our https://github.com/hazelcast/hazelcast[Hazelcast repository] and our https://docs.hazelcast.com/hazelcast/latest/clients/java#hide-nav[Java client documentation] to better understand what is possible.
```

Comment:
```suggestion
check out our https://github.com/hazelcast/hazelcast[Hazelcast repository] and our https://docs.hazelcast.com/hazelcast/latest/clients/java[Java client documentation] to better understand what is possible.
```

Reply:
> Thanks but I'm not sure I understand the difference?

The suggested change above ^ 
We don't need the #hide-nav on the end of the url - think it's an anachronism.

Reply:
> The suggested change above ^ We don't need the #hide-nav on the end of the url - think it's an anachronism.

Sorry, I thought you were suggesting adding it, I didn't realise it was already there. That page has gone from upstream now anyway so a non-issue.